includes/class-wc-countries.php

```diff
@@ -377,6 +376,26 @@ public function countries_using_vat() {
 		return apply_filters( 'woocommerce_countries_using_vat', $countries );
 	}
 
+	/**
+	 * Gets an array of countries using VAT.
+	 *
+	 * @since 4.0.0
+	 * @return array of country codes.
+	 */
+	public function get_vat_countries() {
```

Comment:
I think this creates a confusing API for WC extensions:
- There is a public function `countries_using_vat` with an array of non-EU countries, but excludes some non-EU countries like Norway or GB
- This is another public function `get_vat_countries` which uses `countries_using_vat`, but also adds EU countries and an array of a couple more countries that also use VAT that geographically lie in Europe… 

Function names don't really help to know which function does what, only the comment is explaining it. 

Additionally, if we're going for the split of EU vs non-EU countries, why are GB, IM, MC and NO excluded from the first function? While they're geographically in Europe, they are not EU countries.

Reply:
@peterfabian I'm just trying to make both PRs work together. But I can remove some of the methods.

Reply:
I solved by deprecating `countries_using_vat()`.
So now we can only use `$this->get_vat_countries()` and stop using `array_merge()` for include more countries.